app/views/transport/preRequisites/ContainerIndicatorView.scala.html

```diff
@@ -41,10 +41,9 @@
 
         @inputYesNo(
             field = form("value"),
-            yesNoType = YesNoWithAdditionalHtml(
+            yesNoType = OrdinaryYesNo(
                 heading = messages("transport.preRequisites.containerIndicator.heading"),
-                caption = Some(messages("transport.caption")),
-                additionalHtml = messages("transport.preRequisites.containerIndicator.paragraph").toParagraph
+                caption = Some(messages("transport.caption"))
```

Comment:
Can the `transport.preRequisites.containerIndicator.paragraph` key be removed from `messages`?